tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerDelegate.java

```diff
@@ -91,23 +87,154 @@ public IStatus canModifyModules( IModule[] add, IModule[] remove )
         return retval;
     }
 
+    public String getExternalProperties()
+    {
+        return getAttribute( PROPERTY_EXTERNAL_PROPERTIES, StringPool.EMPTY );
+    }
+
+    @Override
+    public String getHttpPort()
+    {
+        return "8080";
```

Comment:
Lets add a TODO and create a ticket for making this read from PortalBundle class instead.